pysat/tests/test_utils.py

```diff
@@ -21,6 +21,109 @@
 from pysat import utils
 
 
+class TestUpdateFill(object):
+    """Tests for the core utility `update_fill_values`."""
+
+    def setup_method(self):
+        """Set up the test enviroment."""
+        self.ref_time = pysat.instruments.pysat_testing._test_dates['']['']
+        self.new_fill_val = -47.0
+        return
+
+    def teardown_method(self):
+        """Clean up the test environment."""
+        del self.ref_time, self.new_fill_val
+        return
+
+    @pytest.mark.parametrize("name", ["ndtesting", "testing", "testing_xarray",
+                                      "testmodel"])
```

Comment:
```suggestion
    @pytest.mark.parametrize("name", ["ndtesting", "testing", "testmodel"])
```
Since testing_xarray is deprecated, this should cover the code space.  Or add a TODO above this line linked to the issue for when we remove that instrument.